tensorrt_llm/executor/worker.py

```
@@ -343,13 +343,14 @@ def start(self):
         if mpi_rank() == 0:
             self.start_thread(self.dispatch_stats_thread)
 
-    def _load_lora_adapter(self, lora_request: LoRARequest):
-        self._lora_manager.load_from_ckpt(
+    def _load_lora_adapter(self, lora_request: LoRARequest) -> bool:
+        uids = [str(lora_request.adapter_id)]
+        return uids == self._lora_manager.load_from_ckpt(
```

Well, might be a nit, but in terms of SW, I don't like the this `==` clause. I think you should either throw an exception (and catch here) if none was loaded, or use another layer of abstraction.  
It's debatable though.

---

If none were loaded it just means they're not new, which is OK, so far this method just did nothing on that flow. I just want it to return whether something was actually loaded or not

---

Updated the function, hopefully it's clearer now, also added a short docstring about its return value.